redbot/core/core_commands.py

```diff
@@ -3633,8 +3633,22 @@ async def _set_api(
         else:
             if ctx.bot_permissions.manage_messages:
                 await ctx.message.delete()
+            embed = discord.Embed()
+            for api_service_name, token in tokens.items():
+                if token.startswith("<") and token.endswith(">"):
+                    angle_bracket_warning = (
+                        "You may have failed to properly format your {api_service_name}. If you were told to enter a key"
+                        " with an example such as [p]set api {service} api_key <your_api_key_here>, and your API key"
+                        " was HREDFGWE, make sure to run [p]set api {service} api_key HREDFGWE, and not "
+                        " [p]set api {service} api_key <HREDFGWE>"
+                    ).format(api_service_name=api_service_name, service=service)
+                    log.warning(angle_bracket_warning)
+                    embed.add_field(name=_("Warning"), value=_(angle_bracket_warning))
             await ctx.bot.set_shared_api_tokens(service, **tokens)
-            await ctx.send(_("`{service}` API tokens have been set.").format(service=service))
+            await ctx.send(
+                _("`{service}` API tokens have been set.").format(service=service),
+                embed=embed if len(embed.fields) > 0 else None,
```

Comment:
It isn't necessary to give this warning inside of an embed, when there is already text content (it would make things more complicated, such as having to check if an embed is requested in the current channel). Instead, you could add line breaks under the original message and put the warning text there. Something like this would suffice:
```py
angle_bracket_warning = None # define this before iterating through tokens
for api_service_name, token in tokens.items():
    ...
```
```py
message = _("`{service}` API tokens have been set.").format(service=service)
if angle_bracket_warning:
    message += "\n\n" + _("**Warning:** ") + _(angle_bracket_warning)
await ctx.send(message)
```

Remember to remove the `embed` definition on line 3636.